### PR TITLE
#160578719 Apply review to place new order

### PR DESCRIPTION
**Pull request by daniellamarr**
#### What does this PR do?
- This PR applies changes to api route for placing an order
#### Description of Task to be completed?
- When a **POST** request is sent to **localhost:6060/api/v1/orders**, an order should be placed on the platform
- The request returns a status error of **400** when either of **order**, **quantity** or **price** are null or empty
- A successful request should return a status of 201
#### How should this be manually tested?
- Pull this branch **ft-apply-reviews-place-new-order-160578719** off this repository
- On your command line, run npm install to install all dependencies for this app
- On your command line, run npm start to start the app
- App would run on port **6060**
- Access the endpoint with a **POST** request on **localhost:6060/api/v1/orders**
#### Any background context you want to provide?
- This app is a **node app** and would require **nodejs** and **npm** installation on your local machine
#### What are the relevant pivotal tracker stories?
[160578719](https://www.pivotaltracker.com/story/show/160364109)
#### Screenshots (if appropriate)
Request Object
placeorder1
Response Object
placeorder2
Error Object when a field is empty or null
placeorder3

#### Questions:
None

### CODE REVIEW

File: server/routes/route.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import orderControl from "../controls/orderControl";
+import orderControl from "../controllers/orderControl";
+// import Middleware from "../helpers/middleware";
```
Comment (fob413) [inline server/routes/route.js]: why do you have this commented out?

File: server/model/orders.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const orders = [
+    {
+        id:1,
+        menu:[
+            {id:1,order:"Meatpie",quantity:2,price:2000},
+            {id:2,order:"Shawarma",quantity:4,price:4000},
+            {id:3,order:"Pizza",quantity:1,price:1000}
+        ],
+        total: 7000,
```
Comment (fob413) [inline server/model/orders.js]: Please calculate this very well. Total cannot be 7000

**Review by fob413 — CHANGES_REQUESTED** [review]
Delete the `data` and `controls` directories if they're no longer in use

File: server/model/orders.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const orders = [
             {id:2,order:"Shawarma",quantity:4,price:4000},
```
Comment (fob413) [inline server/model/orders.js]: Please update the prices here to reflect the price of an individual item. The total is right.

**Review by fob413 — CHANGES_REQUESTED** [review]
placing an order with an empty `request.body` seems to break your app. Please test this edge case and fix

**Review by fob413 — CHANGES_REQUESTED** [review]
Remove all your validations from your controllers to middleware.

**Review by fob413 — CHANGES_REQUESTED** [review]
sending a request with `menu` which doesn't hold anything seems to break your app. Please validate this edge case

**Review by fob413 — APPROVED** [review]
Go ahead and merge
